rust/gui-client/src-tauri/src/bin/firezone-gui-client.rs

```diff
@@ -145,8 +145,12 @@ fn run_gui(config: RunConfig) -> Result<()> {
             }
 
             if anyhow.root_cause().is::<gui::AlreadyRunning>() {
+                return Ok(());
```

Comment:
Detecting an already running instance is now no longer an error for the 2nd client.